spatial.js

```diff
@@ -0,0 +1,27 @@
+if (typeof (Number.prototype.toRad) === 'undefined') {
+  Number.prototype.toRad = function () {
+    return this * Math.PI / 180;
+  };
+}
+
+module.exports = {
+  distance: distance
+};
+
+function distance(start, end) {
```

Comment:
i was going to including things like bearings to build walk paths, which i couldn't find a single lib for, seem unnecessary to include multiple libs to do a few spatial functions.